markup.go

```diff
@@ -170,7 +170,19 @@ func If(cond bool, markup ...MarkupOrComponentOrHTML) MarkupOrComponentOrHTML {
 	if cond {
 		return List(markup)
 	}
-	return nil
+	var result []MarkupOrComponentOrHTML
+	for _, m := range markup {
+		if m == nil {
+			continue
+		}
+		switch m.(type) {
+		case *HTML, Component:
+			result = append(result, Tag("noscript"))
```

Comment:
Vecty performs this translation in the case of a component `Render` return value being nil.  Possibly we could handle this within `List.Apply()` instead of `If` though, which would give better coverage.  I couldn't see a sane way of incorporating this into `apply()` in `markup.go` though.